src/Language/PureScript/Docs/RenderedCode/Render.hs

```diff
@@ -0,0 +1,172 @@
+
+-- | Functions for producing RenderedCode values from PureScript Type values.
+
+module Language.PureScript.Docs.RenderedCode.Render (
```

Comment:
I thought about doing this. Composing `renderType` with `codeToString` from the `AsMarkdown` module gets you most of the way there, there are just a couple of things that are not handed (eg `Skolem`).